libtransmission/torrent.c

```diff
@@ -3378,6 +3378,58 @@ void tr_torrentSetLocation(tr_torrent* tor, char const* location, bool move_from
     tr_runInEventThread(tor->session, setLocation, data);
 }
 
+char const* tr_torrentPrimaryMimeType(tr_torrent const* tor)
+{
+    struct count
+    {
+        size_t length;
```

Comment:
Denotes file size, `size_t` is not suitable.
```suggestion
        uint64_t length;
```